specification/powerplatform/resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/subnetResources.json

```diff
@@ -0,0 +1,311 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-10-30-preview",
+    "title": "PowerPlatform"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerPlatform/enterprisePolicies/{enterprisePolicyName}/subnets/{subnetName}": {
+      "get": {
+        "tags": [
+          "Subnets"
+        ],
+        "summary": "Gets Subnets associated with EnterprisePolicy authorized for outbound calls from PowerPlatform.",
+        "description": "Description of Subnets that are authorized for outbound calls from PowerPlatform.",
+        "operationId": "Subnets_Get",
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/EnterprisePolicyNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubnetNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Subnet"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets subnet.": {
+            "$ref": "./examples/subnetGet.json"
+          }
+        },
+        "x-ms-long-running-operation": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ]
+      },
+      "put": {
+        "tags": [
+          "PowerPlatform"
+        ],
+        "summary": "Create or Update Subnet that PowerPlatform resources can access.",
+        "description": "Description of the Subnet that PowerPlatform resources can access.",
+        "operationId": "Subnets_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
```

Comment:
>../../common/v1/definitions.json [](start = 21, length = 32)

Suggest referring the common file [https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json) instead of copying the common fields.

Reply:
done